src/app/modules/organisations/organisations-list/organisations-list.component.html

```diff
@@ -52,19 +52,22 @@
           Bulk upload users
         </button>
       </div> -->
-      <div class="btn-gap">
+      <!-- <div class="btn-gap">
         <button (click)="downloadOrganisations()" mat-button matTooltip="Download Organisations">
           Download Organisations
         </button>
-      </div>
+      </div> -->
     </div>
   </div>
 
   <div class="example-container mat-elevation-z8">
     <app-spinner *ngIf="listing == false"></app-spinner>
-    <mat-table [dataSource]="dataSource" matSort>
 
-      <ng-container matColumnDef="select">
+
+
+    <mat-table *ngIf="dataSource" [dataSource]="dataSource" matSort>
+
+      <!-- <ng-container matColumnDef="select">
         <th mat-header-cell *matHeaderCellDef style="padding-top: 10px;">
           <mat-checkbox (change)="$event ? masterToggle() : null" [checked]="selection.hasValue() && isAllSelected()"
```

Comment:
Remove all commented code if not required

Reply:
We are not sure what is required, that is why it is commented ,after discussion i will remove completely